### ISSUE TITLE
Some applications speak links etc.

### ISSUE THREAD

**Issue by tomnet** [pilot51/voicenotify#6]
Some apps for (example "ZDF Mediathek") start to behave like Talkback would be activatet (announcing links by voice etc.) Some other apps show notification that are not announced by voicenotify. (for example "SBB Mobile")
Tested on an xperia Z.
I've only activated VoiceNotify in the accesibility settings,NOT TalkBack.
